tests/main.test.ts

```diff
@@ -77,6 +78,54 @@ describe("User tests", () => {
     });
   });
 
+  it("Should accurately calculates prices", () => {
+    const context = {
+      config: {
+        basePriceMultiplier: 3.0,
+      },
+    };
+    const priority1 = "1 priority";
+    const priority2 = "2 priority";
+    const priority3 = "3 priority";
+    const testCases = [
+      {
+        timeValue: calculateLabelValue("<1 minutes"),
+        priorityValue: calculateLabelValue(priority3),
+        expectedPrice: "1.8",
+      },
+      {
+        timeValue: calculateLabelValue("<4 hours"),
+        priorityValue: calculateLabelValue(priority2),
+        expectedPrice: "300",
+      },
+      {
+        timeValue: calculateLabelValue("<1 hours"),
+        priorityValue: calculateLabelValue(priority2),
+        expectedPrice: "75",
+      },
+      {
+        timeValue: calculateLabelValue("<1.52 hours"),
+        priorityValue: calculateLabelValue(priority3),
+        expectedPrice: "112.5",
+      },
+      {
+        timeValue: calculateLabelValue("<139.876 minutes"),
+        priorityValue: calculateLabelValue(priority1),
+        expectedPrice: "83.4",
+      },
+      {
+        timeValue: calculateLabelValue("<12.333333 weeks"),
+        priorityValue: calculateLabelValue(priority2),
+        expectedPrice: "7800",
+      },
+    ];
+    for (const testCase of testCases) {
+      // @ts-expect-error ignore untyped context;
+      const price = calculateTaskPrice(context, testCase.timeValue, testCase.priorityValue);
```

Comment:
I think you can use `as unknown as Context` to avoid disabling `ts checks`.

Reply:
Appreciate the heads-up!